SafeguardSessions/modules/version/SupportedVersions.pqm

```diff
@@ -0,0 +1,7 @@
+let
+    SupportedVersions.LatestVersion = "7.4.0", SupportedVersions.Versions = {"7.3", "7.4"}

```

Comment:
I don't think this is the right time to add 7.4.0. What if for some reason we need to do a release before 7.4.0 gets out? I know there is almost no chance for it to happen, but never say never.

We can do it later as part of the release process. :slightly_smiling_face:

Reply:
Done